src/markdown-pages/docs/ecosystem/ecosystem-installing-k8s.md

```diff
@@ -0,0 +1,123 @@
+```
+kubectl apply -f deploy/crds/galasa.dev_galasaecosystems_crd.yaml
+```
+The CRD allows the topology and basic configuration for an ecosystem to be defined. A message is displayed to confirm that the CRD is successfully created.
+2. Define the service account by using the <a href=https://github.com/galasa-dev/extensions/tree/master/galasa-ecosystem-kubernetes-operator/deploy/service_account.yaml target="_blank"> service_account.yaml</a> that is provided with Galasa and running the following command:
+```
+kubectl apply -f service_account.yaml
+``` 
+3. Define the role by using the <a href=https://github.com/galasa-dev/extensions/tree/master/galasa-ecosystem-kubernetes-operator/deploy/role.yaml target="_blank"> role.yaml</a> that is provided with Galasa and running the following command:
+```
+kubectl apply -f role.yaml
+```
+4. Define the role binding by using the <a href=https://github.com/galasa-dev/extensions/tree/master/galasa-ecosystem-kubernetes-operator/deploy/role_binding.yaml target="_blank"> role_binding.yaml</a> that is provided with Galasa and running the following command:
+```
+kubectl apply -f role_binding.yaml
+```
+5. Deploy the physical operator by using the <a href=https://github.com/galasa-dev/extensions/tree/master/galasa-ecosystem-kubernetes-operator/deploy/operator.yaml target="_blank"> operator.yaml</a> that is provided with Galasa and running the following command:
+```
+kubectl apply -f deploy/operator.yaml
+```
+6. Check that the pod Galasa operator has come up cleanly by running the ```kubectl get pod``` command. The following service is displayed with a status of *Running*:
+```
+NAME                                                        READY   STATUS    
+galasa-ecosystem-kubernetes-operator-6cb9d79fb5-7zn6f       1/1     Running   
+```
+The operator and custom resource definitions are now installed and ready to bring up a Galasa Ecosystem. 
+
+## Bringing up the Galasa Ecosystem
+
+Bring up the ecosystem by using the <a href=https://github.com/galasa-dev/extensions/blob/master/galasa-ecosystem-kubernetes-operator/deploy/crds/galasa.dev_v1alpha1_galasaecosystem_cr.yaml target="_blank"> custom resource sample</a> that is provided with Galasa.
+
+
+1. Set the ```externalhostname``` value in the sample to the IP address or hostname of your Kubernetes cluster. The Kubernetes Operator needs this information to configure the Galasa Ecosystem to self-register services. 
+2. Update any other default configurations required for the sample to work with your cluster and ensure that the Galasa version number in the sample is the latest version number.
+3. Install the sample by running the following command:
+```
+kubectl apply -f deploy/crds/galasa.dev_v1alpha1_galasaecosystem_cr.yaml
+```
+The installation takes a few minutes. 
+4. Check the status by running the following command:
+```
+kubectl get GalasaEcosystem
+```
+The following example shows the information that is returned by running the command: 
+```
+NAME : 
```

Comment:
The format of this doesnt look good. Its not what would be expected on the command line